tests/unit_tests/Fitting/minimizers/test_minimizer_bumps.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+
+from unittest.mock import MagicMock
+import numpy as np
+
+import easyscience.fitting.minimizers.minimizer_bumps
+from easyscience.Objects.new_variable import Parameter
+
+from easyscience.fitting.minimizers.minimizer_bumps import Bumps
+from easyscience.fitting.minimizers.utils import FitError
+
+
+class TestBumpsFit():
+    @pytest.fixture
+    def minimizer(self) -> Bumps:
+        minimizer = Bumps(
+            obj='obj',
+            fit_function='fit_function', 
+            method='scipy.leastsq'
+        )
+        return minimizer
+
+    def test_init(self, minimizer: Bumps) -> None:
+        assert minimizer._p_0 == {}
+        assert minimizer._cached_pars_order == ()
+        assert minimizer.wrapping == 'bumps'
+
+    def test_init_exception(self) -> None:
+        with pytest.raises(FitError):
+            Bumps(
+                obj='obj',
+                fit_function='fit_function', 
+                method='not_leastsq'
+            )
+
+    def test_available_methods(self, minimizer: Bumps) -> None:
+        # When Then Expect
+        assert minimizer.available_methods() == ['amoeba', 'de', 'dream', 'newton', 'scipy.leastsq', 'lm', 'pt']
+
```

Comment:
The `pt` method in Bumps is experimental and shouldn't really be used. Also, why is `scipy` here?

Reply:
OK, we will then have to filter the raw output from bumps.

I guess it is called `scipy.leastsq` because it utilizes the scipy implementation.  From the bumps code it appears that  `LevenbergMarquardtFit` is called scipy.leastsq.  Should we rename ?